docs/UserGuide.md

```diff
@@ -528,25 +670,7 @@ Format: `delete-attendance c/CLASS_INDEX l/LESSON_INDEX s/STUDENT_INDEX w/WEEK`
 Examples:
 * `delete-attendance c/1 l/1 s/1 w/1` Deletes the 1st week's attendance record of the 1st student of the 1st lesson of the 1st class.
 
-### Undo previous command : `undo`
-
-Undo most recent command.
-
-Format: `undo`
-
-### Redo an undo : `redo`
-
-Redo the most recent undo command.
-
-Format : `redo`
-
-### View action history : `view-history`
-
-View a list of actions that can be undone or redone.
-
-Format : `view-history`
-
-### Statistics : `stats`
+#### Statistics : `stats`
```

Comment:
Perhaps it will be better to change to `Display Statistics` so that it is consistent with the summary table on top?
```suggestion
#### Display Statistics : `stats`
```